tests/test_main.py

```diff
@@ -1596,6 +1596,18 @@ class Config:
         r.id = 2
 
 
+def test_repr_field():
+    class Model(BaseModel):
+        a: int = Field()
+        b: int = Field(repr=True)
+        c: int = Field(repr=False)
+
+    m = Model(a=1, b=2, c=3)
+    assert 'a=1' in repr(m)
+    assert 'b=2' in repr(m)
+    assert 'c=3' not in repr(m)
```

Comment:
```suggestion
    assert 'a=1' in repr(m)
    assert 'b=2' in repr(m)
    assert 'c=3' not in repr(m)
```
```suggestion
    assert repr(m) == 'Model(a=1, b=2)'
    assert repr(m.__fields__['a'].field_info) == 'FieldInfo(default=Ellipsis, extra={})'
    assert repr(m.__fields__['b'].field_info) == 'FieldInfo(default=Ellipsis, extra={})'
    assert repr(m.__fields__['c'].field_info) == 'FieldInfo(default=Ellipsis, repr=False, extra={})'
```
to also test the `field_defaults_to_hide` part

Reply:
Good point 👍 .